src/main/java/seedu/address/logic/parser/ViewCommandParser.java

```diff
@@ -58,7 +59,7 @@ public ViewCommand parse(String args) throws ParseException {
         if (type.equals(VIEW_PERSON_ARG_WORD)) {
             return new ViewPersonCommand(index);
         } else {
-            return new ViewPersonCommand(index); // placeholder until companies are created
+            return new ViewCompanyCommand(index); // placeholder until companies are created
```

Comment:
Since companies have been implemented, perhaps we should remove this comment?

Reply:
Sure. The comment can be removed.